src/libs/actions/Session/index.js

```diff
@@ -646,6 +647,73 @@ function authenticatePusher(socketID, channelName, callback) {
     });
 }
 
+/**
+ * Request a new validation link / magic code to unlink an unvalidated secondary login from a primary login
+ */
+function requestUnlinkValidationLink() {
+    const optimisticData = [{
+        onyxMethod: CONST.ONYX.METHOD.MERGE,
```

Comment:
FYI this just got removed! And it was replaced with `Onyx.METHOD.MERGE`, where `Onyx` is imported from `react-native-onyx`.

I can spin up a fix for this real quick

Reply:
Oh I guess this was found in this bug report: https://github.com/Expensify/App/issues/18526